services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/MessageWriterTest.java

```diff
@@ -113,6 +112,7 @@ void testQueueMessage() {
 
             messageWriter.queueMessage(sdtMessage, UNIT_TEST, false);
             assertTrue(true, SUCCESS);
```

Comment:
Personally, I'm not keen on this sort of check - it will always pass which could lead to false results.  I think it's been included so that the test has at least one assertion which would otherwise be picked up as a code smell.

I'd replace this with a check of something that is done by queueMessage() such as setting the timestamp on the sdtMessage object.